### ISSUE TITLE
Implement remote brick control using FUSE [feature request]

### ISSUE THREAD

**Issue by JakubVanek** [ev3dev/ev3dev#1060]
### What are you trying to do?

It might be interesting to implement a FUSE overlay on /sys to allow remote control of the brick over network.

### What did you expect to happen?

Non-compiled programs on PC would run almost the same way they run on the brick. It might also help with testing language bindings, e.g. https://github.com/ev3dev-lang-java/ev3dev-lang-java/issues/423. The server would just need to be run on the developer machine and instructed to return fake data.

### What actually happened?
Nothing yet.

### What hardware and software are you using (including version numbers)?

None yet.

**Comment by dlech** [comment]
Yes, I've heard from a few people who tried something like this. They reported that it was very slow, but it works.
